dbscripts/xml/upgrade.xml

```diff
@@ -158,6 +158,7 @@
 		<migration class="PKP\migration\upgrade\v3_5_0\I10292_UpdateControlledVocabAssocId"/>
 		<migration class="PKP\migration\upgrade\v3_5_0\I10292_UpdateControlledVocabEntrySettingName"/>
 		<migration class="PKP\migration\upgrade\v3_5_0\FilterClassNames"/>
+		<migration class="PKP\migration\upgrade\v3_5_0\I4860_MigratePublicationVersion"/>
```

Comment:
This should be in the 3.6.0 set of migrations.

Reply:
Changed! (I think)